scripts/release/build.js

```diff
@@ -18,6 +21,22 @@ const verbatimFiles = ['LICENSE', 'README.md', 'index.d.ts', 'types.d.ts'];
 // Returns a string representing data ready for writing to JSON file
 async function createDataBundle() {
   const { default: bcd } = await import('../../index.js');
+
+  const walker = walk(undefined, bcd);
+
+  for (const feature of walker) {
+    for (const [browser, supportData] of Object.entries(
+      feature.compat.support,
+    )) {
+      if (supportData === 'mirror') {
+        feature.data.__compat.support[browser] = bumpData(
```

Comment:
The name `bumpData` doesn't really make sense in this context I think. Maybe `mirrorData` or `mirrorSupport`?